src/Integration.Vsix/CFamily/CFamilyHelper.cs

```diff
@@ -187,58 +187,58 @@ private static Request CreateRequest(FileConfig fileConfig, string absoluteFileP
             var defaultSeverity = rulesConfiguration.RulesMetadata[cfamilyIssue.RuleKey].DefaultSeverity;
             var defaultType = rulesConfiguration.RulesMetadata[cfamilyIssue.RuleKey].Type;
 
-            return new Sonarlint.Issue()
-            {
-                FilePath = cfamilyIssue.Filename,
-                Message = cfamilyIssue.Text,
-                RuleKey = sqLanguage + ":" + cfamilyIssue.RuleKey,
-                Severity = Convert(defaultSeverity),
-                Type = Convert(defaultType),
-                StartLine = cfamilyIssue.Line,
-                EndLine = cfamilyIssue.EndLine,
+            return new AnalysisIssue
```

Comment:
This is code that converted from a CFamily protobuf issue to a daemon protobuf issue. It now converts to an `IAnalysisIssue`.